polaris-react/src/components/DropZone/DropZone.scss

```diff
@@ -122,7 +121,7 @@ $dropzone-stacking-order: (
 }
 
 .Container {
-  @include focus-ring($border-width: var(--p-border-width-1));
```

Comment:
These "`border-width`" values for focus ring are also offsets...and sometimes are only used an an offset so it seemed strange to use `border-width` tokens for them.